package.json

```diff
@@ -52,7 +52,7 @@
         "uuid": "^8.3.2"
     },
     "devDependencies": {
-        "ava": "^0.25.0",
+        "ava": "^3.15.0",
```

Comment:
lol

Reply:
cool, I tried to upgrade before but failed miserably. Looks like switching to `require`  might have done the trick?

Reply:
Yup, there were a few other breaking changes in the library.